core/ir/aarch64/codec.c

```diff
@@ -9697,7 +9697,7 @@ encode_common(byte *pc, instr_t *i, decode_info_t *di)
     ASSERT(((ptr_int_t)pc & 3) == 0);
 
 #if defined(DR_HOST_NOT_TARGET) || defined(STANDALONE_DECODER)
-    dr_set_sve_vl(256);
+    dr_set_sve_vector_length(256);
```

Comment:
Still not understanding this: is this trying to be a default value for cases where the target is not the underlying hardware?  This then misses the case of libdynamorio.so used as a standalone library ("standalone mode": see https://dynamorio.org/page_standalone.html#sec_standalone which talks about two different options; STANDALONE_DECODER is only set for one of those).  Can this just be a static initial value for the internal vector length if nobody set it and we remove this from here?

A related question: someone using the drdecode library might set the vector length to a particular value.  This line would then ignore what they asked for and use 256 when encoding?  But the length they asked for would be used when decoding?  That does not seem right.  Am I missing something?

Reply:
> Am I missing something?

No, my bad! This is a redundant leftover from a previous change which should have been removed! Deleted.